### ISSUE TITLE
The loss displayed by the network fit is inconsistent with the loss calculated from the input data after the model is saved.

### ISSUE THREAD

**Issue by ewwll** [keras-team/keras#16834]
When using keras for regression problems, loss uses mean squared error. The loss displayed by the network fit in the last epoch during training is inconsistent with the loss calculated from the input data after the model is saved.

**Comment by sushreebarsa** [comment]
@ewwll Could you please confirm if it is a duplicate of [this](https://github.com/keras-team/keras/issues/16836) ticket? 
Thank you!
